{"uuid": "73c8e221-f9c0-4864-8f9a-c7164b565cde", "children": ["896cf602-df3a-4843-a0e7-5944f352f855", "ed1ecd6a-04d0-4184-9064-f0e606509a77", "e03ce145-0ec4-43ca-ba31-42a5f79b04b6", "48d010c3-c312-43c8-b1b3-b1da2ef784be", "7a6b755b-6aa9-4636-b624-84114263c497"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586835998129, "stop": 1586835998129}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586836000568, "stop": 1586836008745}], "start": 1586835998129, "stop": 1586836008745}